Tests/DependencyInjection/Compiler/JMSFormErrorHandlerPassTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * This file is part of the FOSRestBundle package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\RestBundle\Tests\DependencyInjection\Compiler;
+
+use FOS\RestBundle\DependencyInjection\Compiler\JMSFormErrorHandlerPass;
+use JMS\Serializer\Handler\FormErrorHandler as JMSFormErrorHandler;
+use FOS\RestBundle\Serializer\Normalizer\FormErrorHandler;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class JMSFormErrorHandlerPassTest extends TestCase
+{
+    private $param = 'jms_serializer.form_error_handler.class';
+
+    public function testParameterNotSetWhenParameterNotFound()
+    {
+        $container = $this->getMockBuilder(ContainerBuilder::class)->getMock();
```

Comment:
please do not mocker the DI related classes

Reply:
I used the other CompilerPass tests as template. Shoud I use a real instance of ContainerBuilder?